extensions/core/interfaces/map/display.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="readonly-map">
+    <div class="interface-map-wrap">
+      <div class="map-display" id="directusMapDisplay" :style="{width:options.width+'px',height:options.height+'px'}">
+        <!-- Map Renders Here -->
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import mapMixin from "./map.js";
+
+export default {
+  name: "readonly-map",
+  mixins: [mapMixin],
+  data() {
+    return {
+      mapPlaceholder: "directusMapDisplay",
+      mode: "display"
+    };
+  },
+  watch: {
+    //? Do we need to re-render map when "value" changes from interface debugger?
```

Comment:
Yes. The value can be set by other interfaces as well, not just the interface debugger!

Reply:
As we do not display map, I don't need to rerender map & update marker. So I'm removing this watcher. Right?